src/components/Disputes/DisputeEvidences.js

```diff
@@ -154,16 +155,30 @@ const EvidenceContent = React.memo(function EvidenceContent({
   )
 })
 
-function getSubmitterLabel(submitter, dispute) {
-  if (addressesEqual(submitter, dispute.defendant)) {
+function getSubmitterLabel(submitter, defendant, plaintiff) {
+  if (addressesEqual(submitter, defendant)) {
     return 'Defendant'
   }
 
-  if (addressesEqual(submitter, dispute.plaintiff)) {
+  if (addressesEqual(submitter, plaintiff)) {
     return 'Plaintiff'
   }
 
   return ''
 }
 
-export default DisputeEvidences
+export default ({ dispute, evidences }) => {
+  const [evidenceProcessed, fetchingEvidences] = useEvidences(
```

Comment:
The hook ensures us that `evidenceProcessed` won't be updated unless there are new evidences.